src/AutoRest.CSharp/Common/Input/InputTypes/Examples/Serialization/TypeSpecInputExampleValueConverter.cs

```diff
@@ -155,11 +155,19 @@ static InputExampleValue ParseUnknownValue(InputType type, JsonElement? rawValue
                     case JsonValueKind.True or JsonValueKind.False:
                         return InputExampleValue.Value(type, rawValue.Value.GetBoolean());
                     case JsonValueKind.Number:
-                        if (rawValue.Value.TryGetInt32(out var int32Value))
+                        var rawText = rawValue.Value.GetRawText();
+                        if (int.TryParse(rawText, out var int32Value))
```

Comment:
so `rawValue.Value.TryGetfloat` doesn't work, but `float.TryParse(rawText, out var floatValue)` works?

Reply:
I somehow cannot reproduce this in a standalone program now - but last time when I debug this, `rawValue.Value.TryGetSingle` is returning `false`, and `float.TryParse` returns me true.
Maybe there is a complicated condition to reproduce this issue, but the new way definitely will work as expected.